### PR TITLE
Release 1.2.0 beta.0

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO

- [ ] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
TODO: Need to remove Search UI from Discover Feed (or implement search)

**Comment by redreceipt** [comment]
💪

--
Michael Neeley
Junior Web Developer
NewSpring Church
www.newspring.cc
Cell: (803) 622-6599

File: packages/apollos-church-api/config.yml [hunk 1]
```diff
@@ -131,6 +137,10 @@ ROCK_MAPPINGS:
 
   SERMON_CHANNEL_ID: 19
 
+  SERVING_GROUP_TYPE_ID: 23
```
Comment (redreceipt) [inline packages/apollos-church-api/config.yml:140]: I don't think this one is right, the other two are though.

@briankalwat do you know what our group type ID for serving groups is?

File: packages/newspringchurchapp/ios/Podfile [hunk 2]
```diff
@@ -1,9 +1,61 @@
-# Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
 target 'newspringchurchapp' do
-  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
-  # use_frameworks!
+  # Pods for apolloschurchapp
```
Comment (redreceipt) [inline packages/newspringchurchapp/ios/Podfile]:
```suggestion
  # Pods for newspringchurchapp
```

File: packages/newspringchurchapp/ios/Podfile [hunk 3]
```diff
@@ -1,9 +1,61 @@
...
+  pod 'React-RCTBlob', :path => '../node_modules/react-native/Libraries/Blob'
+  pod 'React-RCTImage', :path => '../node_modules/react-native/Libraries/Image'
+  pod 'React-RCTLinking', :path => '../node_modules/react-native/Libraries/LinkingIOS'
+  pod 'React-RCTNetwork', :path => '../node_modules/react-native/Libraries/Network'
+  pod 'React-RCTSettings', :path => '../node_modules/react-native/Libraries/Settings'
+  pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
+  pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
+  pod 'React-RCTWebSocket', :path => '../node_modules/react-native/Libraries/WebSocket'
 
-  pod 'Amplitude-iOS', '~> 3.14.1'
```
Comment (redreceipt) [inline packages/newspringchurchapp/ios/Podfile:8]: @vinnyjth know why this was removed?

**Comment by redreceipt** [comment]
it works!!  we can make PRs for Amplitude and group prayers if they don't work (see above)

**Review by redreceipt — APPROVED** [review]

File: packages/newspringchurchapp/ios/Podfile (hunk 3 above)
Comment (redreceipt) [reply packages/newspringchurchapp/ios/Podfile:8]: app doesn't crash and there's another version in the `Podfile.lock` so I guess it's ok 🤷‍♂

File: packages/apollos-church-api/config.yml (hunk 1 above)
Comment (briankalwat) [reply packages/apollos-church-api/config.yml:140]: @redreceipt there are many serving group types. KS vol, Fuse vol, etc. Do you want all of them?

**Pull request merged**